resources/istio/README.md

```diff
@@ -53,26 +53,10 @@ The chart deploys Pods that consume minimum resources as specified in the resour
     $ kubectl create ns $NAMESPACE
     ```
 
-1. If you are enabling `kiali`, you need to create the secret that contains the username and passphrase for `kiali` dashboard:
-    ```
-    $ echo -n 'admin' | base64
-    YWRtaW4=
-    $ echo -n '1f2d1e2e67df' | base64
-    MWYyZDFlMmU2N2Rm
-    $ cat <<EOF | kubectl apply -f -
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: kiali
-      namespace: $NAMESPACE
-      labels:
-        app: kiali
-    type: Opaque
-    data:
-      username: YWRtaW4=
-      passphrase: MWYyZDFlMmU2N2Rm
-    EOF
-    ```
+1. If you are enabling `kiali`, the secret will be created automatically with the username `admin` and an autogenerated passphrase. To get the password use:
```

Comment:
What is with monitoring? Is it required to get kiali working? If so, please mention it here.

Reply:
Yes, it is. As written in the PR message, Kiali requires Prometheus to be installed.